tests/unit/test_openvswitch.py

```diff
@@ -283,6 +294,28 @@ def test_ovs_bfd_state_changes(self):
         inst = event_checks.OVSEventChecks()
         self.assertEqual(self.part_output_to_actual(inst.output), expected)
 
+    @mock.patch('hotsos.core.ycheck.engine.YDefsLoader._is_def',
+                new=utils.is_def_filter('ovn/ovn-central.yaml'))
+    @utils.create_data_root({'var/log/ovn/ovsdb-server-sb.log':
+                             LEADERSHIP_TRANSFERS},
+                            copy_from_original=['sos_commands/date/date'])
+    def test_ovn_ovsdb_leadership_changes(self):
+        expected = {'ovsdb-server-sb': {'leadership-transfers': {
+                                            '2022-07-27': 2}}}
+        inst = event_checks.OVNEventChecks()
+        self.assertEqual(self.part_output_to_actual(inst.output), expected)
+
+    @mock.patch('hotsos.core.ycheck.engine.YDefsLoader._is_def',
+                new=utils.is_def_filter('ovn/ovn-central.yaml'))
+    @utils.create_data_root({'var/log/ovn/ovsdb-server-sb.log':
+                             DB_COMPACTION},
+                            copy_from_original=['sos_commands/date/date'])
+    def test_ovn_ovsdb_compactions(self):
+        expected = {'ovsdb-server-sb': {'compactions': {
+                                            '2022-07-14': 3}}}
+        inst = event_checks.OVNEventChecks()
+        self.assertEqual(self.part_output_to_actual(inst.output), expected)
```

Comment:
You're probably going to update this again, just commenting there are not tests for NBDB yet so I don't forget.

Reply:
fair enough ill add those